src/zenml/cli/__init__.py

```diff
@@ -298,7 +298,46 @@
 command:
 
 ```bash
-zenml container-registry delete
+zenml container-registry delete REGISTRY_NAME
+```
+
+Customizing your Experiment Tracker
+-----------------------------------
+
+Experiment trackers let you track your ML experiments by logging the parameters
+and allowing you to compare between different runs. If you want to use an
+experiment tracker in one of your stacks, you need to first register it:
+
+```bash
+zenml experiment-tracker register EXPERIMENT_TRACKER_NAME \
+    --type=EXPERIMENT_TRACKER_TYPE [--EXPERIMENT_TRACKER_OPTIONS]
```

Comment:
```suggestion
    --flavor=EXPERIMENT_TRACKER_TYPE [--EXPERIMENT_TRACKER_OPTIONS]
```

In case Baris' PR also is getting merged?